java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -272,6 +277,27 @@ public CompletableFuture<String> functionDelete(@NonNull String libName) {
                 FunctionDelete, new String[] {libName}, this::handleStringResponse);
     }
 
+    @Override
+    public CompletableFuture<byte[]> functionDump() {
+        return commandManager.submitNewCommand(
+                FunctionDump, new GlideString[0], response -> handleBytesResponse(response).getBytes());
+    }
+
+    @Override
+    public CompletableFuture<String> functionRestore(byte @NonNull [] payload) {
```

Comment:
consider changing to @NonNull GlideString instead?

Reply:
since it will never be a String, we can stick with byte[].  What do you think @yury?

Reply:
`byte[]` - same as in `restore`